### DISCUSSION TITLE
401 Unauthorized on deployed app

### DISCUSSION THREAD

**Discussion by Tobi-De** [fastapi-users/fastapi-users#642] [Q&A] [answered]
Hi, I have deployed a very simple fastapi app built using fastapi-users, tortoise-orm and postgres. I tested everything on my local machine and everything worked fine. I added a start event to create a verified superuser. I then deployed the app using docker and traefik on a vps, but all the routes that used the ``` current_user``` dependency all returned  a 401, not sure why. I thought this was an issue with the way the app was deployed, I removed traefik from the stack, removed the domain name, try a bunch of stuff, but still get 401 on the routes that were using  ```current_user```. I then deployed the app to heroku to see if that will change anything but I get the same 401 response, every other routes just worked fine. For the   ```current_user``` dependency I'm using ```current_user=fastapi_users.current_user(active=True, verified=False)```. Does anyone have any idea of ​​the problem? I'm probably doing something wrong somewhere but have no idea, any help is welcome, thanks.

**Comment by frankie567** [comment]
Which authentication backend are you using? JWT or Cookie?

**Reply by Tobi-De** [reply]
@frankie567
JWT, I try to verify the superuser on the online version of the app with the request-verify-token route but no token was generated, probably because the user was already verified.

**Comment by Tobi-De** [comment, marked as answer]
I already found the problem, this thread can be deleted
